cicerone/app/src/main/java/com/teampower/cicerone/GeofenceTriggeredActivity.kt

```diff
@@ -20,7 +22,7 @@ import kotlinx.android.synthetic.main.activity_scrolling.toolbar
 import kotlinx.android.synthetic.main.content_geofence_triggered.*
 import kotlinx.coroutines.MainScope
 import kotlinx.coroutines.launch
-import org.threeten.bp.Instant
+import java.time.Instant
```

Comment:
The reason for not using java.time.Instant is because it requires Android 8 (api 26). I found that to be a bit unreasonable, although it's okay for this project if you really want it. This is a backport, and if you don't mind, we can avoid all of those "Requires O lines" (which I don't like as we should rather update the minApi in the gradle file. As it is we are only saying that part of our app won't work for lower apis).

Reply:
At the moment the app in itself is completely broken on anything below Android 10 anyway. I tried installing it on my phone (Android 9) and it just kept glitching out.

Reply:
I didn't chose java.time.Instant by choice, my IDE complained that it couldn't find org.threeten.bp.Intent, so instead it imported the java-version. So I just did what the IDE told me without thinking about it 😆